db.go

```diff
@@ -0,0 +1,72 @@
+package auth
+
+import (
+	"errors"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+	"github.com/sirupsen/logrus"
+)
+
+const DefaultMaxIdleConns = 5
+
+var ErrorNoConnectionString = errors.New("A connection string must be specified on the first call to Get")
+
+type Storer interface {
```

Comment:
not sure if this should be public...kicking the can on this down the road. Could be useful for package users to have access to the db directly (if we trust em)

Reply:
also not sure how happy I am with the `Storer` interface name...would love suggestions, particularly ones that stick with the -er and -or suffix as per convention